lsp-ui-doc.el

```diff
@@ -234,6 +234,12 @@ MODE is the mode used in the parent frame."
          (funcall render-fn string)
        (with-temp-buffer
          (insert string)
+
+         (if (lsp-ui-doc--inline-p)
+             (progn (let ((fill-column lsp-ui-doc-max-width))
+                      (fill-region (point-min) (point-max)))
+                    (buffer-substring (point-min) (point-max))))
```

Comment:
Why do you need `(buffer-substring (point-min) (point-max))`? The return value is not used.

I think you may use: when + fill-region